ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml

```diff
@@ -79,15 +79,15 @@ promotion:
   - operator-registry-base
   - ocs-registry
   - ocs-registry-master
-  name: ocs-4.8
+  name: ocs-4.9
```

Comment:
what is this thing with the 4.9 vs 4.8 again?

Reply:
I believe this is the name of a placeholder ImageStream that gets created in the CI to reference the images it builds for the PR tests. I figured since we had the pattern of master = 4.8 when we created release-4.7, it should be master = 4.9 now that release-4.8 exists. But again, this is just a Ci-internal name.